inference-engine/src/transformations/src/transformations/op_conversions/convert_interpolate1_to_interpolate4.cpp

```diff
@@ -45,7 +46,13 @@ ngraph::pass::ConvertInterpolate1ToInterpolate4::ConvertInterpolate1ToInterpolat
         if (attrsV0.mode == "nearest") {
             attrsV4.mode = ngraph::opset4::Interpolate::InterpolateMode::nearest;
         } else if (attrsV0.mode == "linear") {
-            attrsV4.mode = ngraph::opset4::Interpolate::InterpolateMode::linear;
```

Comment:
Please, add a comment and "TODO" here describing the reason to use "incorrect" interpolation mode.

Reply:
Also, it make sense to create some tickets for CPU/GPU for linear_onnx mode optimization and left their numbers here.

Reply:
It make sense to create tickets for `linear` mode optimization, because CPU has optimized implementation of the `linear_onnx` mode, and the reason of this PR is an absence of optimized version of `linear` mode.

Reply:
Tickets for CPU and GPU are created.

Reply:
Added comment and TODO